octobot_trading/modes/modes_util.py

```diff
@@ -142,7 +142,14 @@ async def convert_asset_to_target_asset(
                 quantity=order_quantity,
                 price=order_price
             )
-            created_orders.append(await trading_mode.create_order(order))
+            initialized_order = await trading_mode.create_order(order)
+            if (
+                isinstance(initialized_order, trading_personal_data.LimitOrder)
+                and initialized_order.trader.exchange_manager.trader.simulate
```

Comment:
```suggestion
                and initialized_order.is_simulated()
```
I don't remember if this method exists

Reply:
right, there is a `simulated` attribute, I update this

Reply:
up